### PR TITLE
Remove reminder after processing + dont fail if message not found

### PR DESCRIPTION
**Pull request by MilenaHristova**

### CODE REVIEW

File: src/Maestro/Maestro.ContainerApp/Queues/QueueProcessors/PullRequestReminderQueueProcessor.cs [hunk 1]
```diff
@@ -40,5 +48,7 @@ protected override async Task ProcessAsyncInternal(PullRequestReminderWorkItem w
         {
             await pullRequestActor.ProcessPendingUpdatesAsync();
         }
+
+        await _database.StringGetDeleteAsync(workItem.Name);
```
Comment (andriipatsula) [inline src/Maestro/Maestro.ContainerApp/Queues/QueueProcessors/PullRequestReminderQueueProcessor.cs]: Line 40: `throw new ArgumentException(...)`
We can throw an exception that the actorid is invalid. do we need to ensure/to delete workItem from DB if the exception is thrown?

**Review by andriipatsula — APPROVED** [review]
LGTM

File: src/Maestro/Maestro.ContainerApp/Queues/QueueProcessors/PullRequestReminderQueueProcessor.cs (hunk 1 above)
Comment (MilenaHristova) [reply src/Maestro/Maestro.ContainerApp/Queues/QueueProcessors/PullRequestReminderQueueProcessor.cs]: good point

Comment (premun) [reply src/Maestro/Maestro.ContainerApp/Queues/QueueProcessors/PullRequestReminderQueueProcessor.cs]: I'd imagine the whole queue processing and actors would need to get hardened against exceptions and a retry mechanism defined. But that would be for production service

Comment (premun) [reply src/Maestro/Maestro.ContainerApp/Queues/QueueProcessors/PullRequestReminderQueueProcessor.cs]: But yeah, reminder deletion should go inside of `finally {}` of some sort

Comment (MilenaHristova) [reply src/Maestro/Maestro.ContainerApp/Queues/QueueProcessors/PullRequestReminderQueueProcessor.cs]: for now I changed it to remove the message in the beginning

**Review by premun — APPROVED** [review]

**Pull request merged**